config/routes.rb

```diff
@@ -25,8 +25,12 @@
   patch '/shelters/:shelter_id/reviews/:review_id', to: 'reviews#update'
   delete '/shelters/:shelter_id/reviews/:review_id', to: 'reviews#destroy'
 
+  get '/pet_applications/new', to: 'pet_applications#new'
```

Comment:
At one point Cory had mentioned putting commented out headers in here describing what each set of crud functionality is for.  Might want to put that back in there soon... "#Pet Application" just for readability